src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/X509Pal.PublicKey.cs

```diff
@@ -51,6 +56,50 @@ public AsymmetricAlgorithm DecodePublicKey(Oid oid, byte[] encodedKeyValue, byte
             }
         }
 
+        private static ECDsa DecodeECDsaPublicKey(byte[] encodedKeyValue)
+        {
+            if (encodedKeyValue.Length < 1 || (AsnTag)(encodedKeyValue[0]) != AsnTag.OCTET_STRING)
+                throw new CryptographicException(SR.InvalidPublicKeyInX509);
+
+            // After the OCTET_STRING tag, the encryptedKeyValue are X, followed by Y, in big-endian format.
```

Comment:
That’s the answer I was looking for with that mail thread I sent out Friday about decoding the ASN1.

Putting out a PR is truly the most effective ping mechanism…